bugfix/PLUGIN-351 Wrangler nested data types

ailegion opened the pull request:
fixed: Wrangler transform should passing-through nested data types
Jira Ticket: https://issues.cask.co/browse/PLUGIN-351

ailegion committed:
fixed: Wrangler transform should passing-through nested data types

CuriousVini reviewed (changes requested):
Please add tests for this fix. Also add the PR to the jira.
How has this fix been tested?

ailegion committed:
added: Unit test for nested record conversion

ailegion commented:
> Please add tests for this fix. Also add the PR to the jira.
> How has this fix been tested?

Added test for the fix and added PR to jira.
Regarding how it was tested - we applied the steps based on notes in jira ticket. Managed to replicate the issue by setting one the the input fields as type record. Once the fix was applied the pipeline succeeded.
